examples/phewas_scatter.html

```diff
@@ -22,40 +22,131 @@
         max-width: 100%;
         box-sizing: border-box;
       }
+      pre {
+        overflow: auto;
+        background-color: #f1f1f1;
+        border-radius: 3px;
+        font-size: 1em;
+      }
+      code {
+        font-size: 1em;
+      }
+      .spacer {
+        padding: 0px 15px 0px 15px;
+      }
+      .required {
+        color: red;
+      }
+      .error {
+        color: red;
+      }
     </style>
 
   </head>
 
   <body>
     <div class="container">
 
-      <h1 style="margin-top: 1em;"><strong>Locuszoom.js</strong></h1>
+      <h1 style="margin-top: 1em;"><strong>LocusZoom.js</strong></h1>
+
+      <h3 style="color: #777">PheWAS Demonstration</h3>
 
-      <h3 style="float: left; color: #777">PheWAS (Scatter) Example</h3>
-      <h6 style="float: right;"><a href="../index.html">&lt; return home</a></h6>
+      <p>
+        A PheWAS visualization displays association p-values between a given genetic variant and multiple phenotypes. 
+        In this demonstration, we are showing single variant association statistics from multiple publicly available meta-analyses of genome-wide association studies (GWAS), stored locally in a database at the Center for Statistical Genetics (University of Michigan.)
+        For the purposes of the demonstration, we chose reasonable groupings of traits into categories, but any arbitrary grouping is possible. 
+      </p>
 
-      <hr style="clear: both;">
+      <p>
+        LocusZoom.js requires a REST API endpoint to retrieve data for the plot. The format of the response is documented <a target=_blank href='http://portaldev.sph.umich.edu/docs/api/v1/index.html#phewas-all-available-results-for-a-given-variant'>in our API documentation</a>, with a brief synopsis also included below.
```

Comment:
Small note: this demo requires a REST endpoint, but it's not the only option. It's actually possible to serve data from a file, or use the `StaticJSON` source type for data embedded in the same page.

Since the current demo is aimed at a specific audience for delivery, it's not an issue for this page. But we do try to keep our other demos generic to help engage potential integrators with other workflows. It's a general note as we build out future tutorials for a broader audience.